tests/maintel/test_warmup_hexapod.py

```diff
@@ -0,0 +1,125 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+
+# import random
+import unittest
+import numpy as np
+
+from lsst.ts import utils
+from lsst.ts import standardscripts
+from lsst.ts.externalscripts.maintel import WarmUpHexapod
+from lsst.ts.idl.enums import Script
+
+np.random.seed(42)
+
+index_gen = utils.index_generator()
+
+logger = logging.getLogger(__name__)
```

Comment:
Maybe instead of creating a logger here, add log to the test case class.. you can do it in the `setUp` method...